website/docs/r/workspace.html.markdown

```diff
@@ -86,6 +86,7 @@ The following arguments are supported:
   Enterprise instance from running plans on pull requests, which can improve
   security if the VCS repository is public or includes untrusted contributors.
   Defaults to `true`.
+* `structured_run_output_enabled` - (Optional) Whether this workspace should show output from Terraform runs using the enhanced UI when available. Setting this to `false` ensures that all runs in this workspace will display their output as text logs. Defaults to `true`.
```

Comment:
Suggestion:

Can we put "_Defaults to 'true'_" At the start? Its helpful to know, as a reader, what type the field is, and if there is a default from the get go. Then reading the "Wether this workspace..." is clear what is enabled from the start.

Reply:
Great point! I cargo-culted this from the line above, will change both.